### ISSUE TITLE
Evaluation Results on the HQF dataset

### ISSUE THREAD

**Issue by pokhrelapar** [intelpro/CBMNet#15]
@intelpro  Could you please explain briefly how to get results for the HQF dataset shown in Table 3?

I am assuming the following. If you could let me know if this sounds right.

1. use tools/preprocess_events.py to get the event voxel grids on the test set.  Do you happen to have the test/train/val split for HQF dataset so that I can replicate the result.

```python 
    
python tools/preprocess_events.py  --dataset_dir HQF_DATASET_DIR --mode 1_TEST

```

2.  I'd assume we would finetune on the HQF dataset rather than retraining from scratch? Are there specific hyperparameters or preprocessing you used for this? Are there any HQF-specific considerations I should take? Would you be willing to share the relevant scripts to help replicate the metrics in Table 3.

Thanks!
